spaces/decentribe/index.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Decentribe",
+  "network": "1",
+  "symbol": "DECENT",
+  "defaultView": "all",
+  "token": "0x900B1998A226CFE238Ba1a14F03eB934C513C20B",
```

You will miss the `strategies` field.
```suggestion
  "token": "0x900B1998A226CFE238Ba1a14F03eB934C513C20B",
  "strategies": [
    {
      "name": "erc20-balance-of",
      "params": {
        "address": "0x900B1998A226CFE238Ba1a14F03eB934C513C20B",
        "symbol": "DECENT",
        "decimals": 18
      }
    }
  ],
```